data/lua/core/_initial.lua

```diff
@@ -107,3 +107,14 @@ wesnoth.set_end_campaign_text = wesnoth.deprecate_api('wesnoth.set_end_campaign_
 		wesnoth.scenario.end_text_duration = d
 	end
 end)
+
+if wesnoth.kernel_type() ~= 'Application Lua Kernel' then
```

Comment:
The C++ has `intf_find_path` for the mapgen and game kernels, but not the application kernel.

Reply:
That's… why this line exists?

Reply:
Oops, I misread `~=` as `==`